flow/utils/aimsun/generate.py

```diff
@@ -685,7 +685,7 @@ def create_meter(model, edge):
     """
     section = model.getCatalog().findByName(edge, model.getType("GKSection"))
     meter_length = 2
-    pos = section.getLanesLength2D() - meter_length
+    pos = section.length2D() - meter_length
```

Comment:
is this a feature in a newer version of Aimsun? Why wasn't it broken before?

Reply:
I think this was always there, but it's a method available to the parent class. Aimsun's documentation was difficult to navigate through. The reason this wasn't broken before is that this network only had single lanes per edge. `getLanesLength2D` returns the sum of the lengths of all lanes, which isn't appropriate for this use case.